src/components/common/balance/BalanceDisplayUSD.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { getRoundedBalanceInFiat } from './helpers';
+
+const BalanceDisplayUSD = ({
+    amount,
+    showAlmostEqualSignUSD = true,
+    showSignUSD = false,
+    showSymbolUSD = true,
+    nearTokenFiatValueUSD
+}) => {
+
+    const roundedBalanceInUSD = getRoundedBalanceInFiat(amount, nearTokenFiatValueUSD);
+    const amountPrefix = roundedBalanceInUSD !== '< 0.01' ? '≈ ' : '';
+    const USDSymbol = 'USD';
+
+    if (roundedBalanceInUSD && roundedBalanceInUSD !== isNaN) {
```

Comment:
Is `isNaN` a global or something? AFAIK the global `isNaN` I'm aware of is a _function_ that must be called.  Also, `getRoundedBalanceInFiat()` returns '< 0.01'.  Wouldn't we want to show that here rather than `- USD`?  If we're just trying to ignore `< 0.01` I'd prefer that we export that magic string from `helpers` and explicitly check it here rather than use `isNaN` or NaN checks which are notoriously difficult to reason about in JS :)

Reply:
Good catch:) Forgot to remove isNaN after refactoring.